app/experimenter/nimbus-ui/src/components/FormOverview/FormDocumentationLink.tsx

```diff
@@ -0,0 +1,99 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+import Button from "react-bootstrap/Button";
+import Col from "react-bootstrap/Col";
+import Form from "react-bootstrap/Form";
+import { ArrayField, FieldError } from "react-hook-form";
+import { useCommonNestedForm, useConfig } from "../../hooks";
+import { ReactComponent as DeleteIcon } from "../../images/x.svg";
+import { AnnotatedDocumentationLink } from "./documentationLink";
+
+export const documentationLinkFieldNames = ["title", "link"] as const;
+
+type DocumentationLinkFieldName = typeof documentationLinkFieldNames[number];
+
+export type FormDocumentationLinkProps = {
+  fieldNamePrefix: string;
+  documentationLink: Partial<ArrayField<AnnotatedDocumentationLink, "id">>;
+  errors: Record<string, FieldError>;
+  onRemove: () => void;
+  canRemove: boolean;
+  submitErrors: Record<string, string[]>;
+  setSubmitErrors: React.Dispatch<React.SetStateAction<Record<string, any>>>;
+  touched: Record<string, boolean>;
+};
+
+export const FormDocumentationLink = ({
+  fieldNamePrefix,
+  documentationLink,
+  errors,
+  onRemove,
+  canRemove,
+  submitErrors,
+  setSubmitErrors,
+  touched,
+}: FormDocumentationLinkProps) => {
+  const { documentationLink: documentationLinkOptions } = useConfig();
+
+  const { FormErrors, formControlAttrs } = useCommonNestedForm<
+    DocumentationLinkFieldName
+  >(
+    {
+      title: documentationLink.title,
+      link: documentationLink.link,
+    },
+    setSubmitErrors,
+    fieldNamePrefix,
+    submitErrors,
+    errors,
+    touched,
+  );
+
+  return (
+    <Form.Group className="mb-0" data-testid="DocumentationLink">
+      <Form.Row>
+        <Form.Group as={Col} sm={4} md={3}>
+          <Form.Control as="select" {...formControlAttrs("title", {})}>
```

Comment:
The 2nd argument along with the 2nd argument on L77 should be removable now if you pull latest.

Reply:
Updated